### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading '$ref')

### ISSUE THREAD

**Issue by jitbasemartin** [anymaniax/orval#247]
## What are the steps to reproduce this issue?

1. Create following specs file `openapi.yaml`

```yaml
openapi: 3.0.0
info:
  title: test
  version: 0.0.1
paths:
  /test:
    get:
      tags:
        - TestController
      responses:
        '200':
          description: Test
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/test'
      parameters:
        - name: where
          in: query
          content:
            application/json:
              schema:
                type: object
                title: test.WhereFilter
                additionalProperties: true
      operationId: test
servers:
  - url: http://localhost:3000/api
components:
  schemas:
    test:
      type: object
      title: test
      properties:
        test:
          type: number
```

2. Run Orval v6.2.4

```sh
npx orval --input ./openapi.yaml --output ./openapi.ts 
```

## What happens?

Got following error

```txt
🍻 Start orval v6.2.4 - A swagger client generator for typescript
🛑  TypeError: Cannot read properties of undefined (reading '$ref')
```

## Comments

The problem is caused by `parameters`, I think Orval expect `$ref` after `content > application/json > schema`

Thanks

**Comment by mwlazlo** [comment]
I also ran into this bug this morning +1

**Comment by tynanspense** [comment]
This is also a problem I'm facing +1

**Comment by anymaniax** [comment]
Should be fixed with `6.4.0` can you test when you have time, please?

**Comment by anymaniax** [comment]
Closing this issue since it's normally fixed. Don't hesitate if you have any problem
